src/window_gui.h

```diff
@@ -692,6 +692,8 @@ struct Window : ZeroedMemoryAllocator {
 
 	/**
 	 * Called once every 100 (game) ticks.
+	 * In normal game speed the frequency is 1 call every 100 ticks.
+	 * In fast-forward the frequency is 1 call every ~3s (can be way more than 100 ticks).
```

Comment:
and in slow-forward? Or is that not worth mentioning here?

Reply:
That's normal game speed